File: Java/bmi.java
import java.util.Scanner;

public class bmi {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter your age: ");
        int age = sc.nextInt();

        System.out.print("Enter your gender (M/F/O): ");
        char gender = sc.next().toUpperCase().charAt(0);

        System.out.print("Enter your weight in kg: ");
        double weight = sc.nextDouble();

        System.out.print("Enter your height in cm: ");
        double heightCm = sc.nextDouble();

    
        double heightM = heightCm / 100;

    
        double bmi = weight / (heightM * heightM);

    
        System.out.println("\nYour Age: " + age);
        System.out.println("Your Gender: " + gender);
        System.out.println("Your BMI: " + String.format("%.2f", bmi));

        if (bmi < 18.5) {
            System.out.println("You're underweight. Eat more nutrient-rich food! 🍕🥗");
        } else if (bmi < 24.9) {
            System.out.println("You're at a healthy weight. Keep up the good work! 💪");
        } else if (bmi < 29.9) {
            System.out.println("You're overweight. Try exercising and eating balanced meals. 🏃‍♂️🍏");
        } else {
            System.out.println("You're in the obese range. Consider a healthier lifestyle! 🏋️‍♂️🥦");
        }

        sc.close(); 
    }
}

File: python/bmi.py
def calculate_bmi(weight, height):
    #BMI Formula: weight (kg) / height^2 (m^2)
    height_m=height / 100
    bmi = weight / (height_m * height_m)
    return round(bmi, 2)
age = int(input("Enter your age: "))
gender = input("Enter your gender (M/F/O): ").strip().upper()
weight = float(input("Enter your weight in kg: "))
height = float(input("Enter your height in centimeters: "))

bmi = calculate_bmi(weight, height)
print(f"\nYour Age: {age}")
print(f"Your Gender: {gender}")
print(f"Your BMI: {bmi}")
if bmi < 18.5:
    print("You're underweight. Eat more nutrient-rich food! 20")
elif bmi < 24.9:
    print("You're at a healthy weight. Keep up the good work! ")
elif bmi < 29.9:
    print("You're overweight. Try exercising and eating balanced meals. ")
else:
    print("You're in the obese range. Consider a healthier lifestyle! ")
